app/src/androidTest/java/org/mozilla/tv/firefox/helpers/FakePocketVideoRepoProvider.kt

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.helpers
+
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.MutableLiveData
+import org.mozilla.tv.firefox.ext.toUri
+import org.mozilla.tv.firefox.pocket.PocketEndpoint
+import org.mozilla.tv.firefox.pocket.PocketFeedStateMachine
+import org.mozilla.tv.firefox.pocket.PocketVideoRepo
+import org.mozilla.tv.firefox.pocket.PocketViewModel
+import org.mozilla.tv.firefox.utils.BuildConfigDerivables
+
+/**
+ * Provides a fake [PocketVideoRepo] implementation for testing purposes.
+ *
+ * Any values pushed to [fakedPocketRepoState] will be immediately emitted.
+ */
+object FakePocketVideoRepoProvider {
```

Comment:
You should make this a class and instantiate it for each test: if the test runner doesn't the class state between tests, this will break and it's also harder to think about.